specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/disasterRecoveryConfigurations.json

```diff
@@ -0,0 +1,405 @@
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the disaster recovery configuration to failover."
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/disasterRecoveryConfiguration/{disasterRecoveryConfigurationName}/forceFailoverAllowDataLoss": {
+      "post": {
+        "tags": [
+          "DisasterRecoveryConfigurations"
+        ],
+        "operationId": "DisasterRecoveryConfigurations_FailoverAllowDataLoss",
+        "description": "Which disaster recovery configuration is being failed-over forcefully. This operation might result in data loss.",
+        "x-ms-examples": {
+          "Failover a disaster recovery configuration": { "$ref": "./examples/DisasterRecoveryConfigurationFailover.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "disasterRecoveryConfigurationName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the disaster recovery configuration to failover forcefully."
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    }
+  },
+  "definitions": {
+    "Resource":{
+      "description":"ARM resource.",
+      "properties":{
+        "id":{
+          "readOnly":true,
+          "type":"string",
+          "description":"Resource ID."
+        },
+        "name":{
+          "readOnly":true,
+          "type":"string",
+          "description":"Resource name."
+        },
+        "type":{
+          "readOnly":true,
+          "type":"string",
+          "description":"Resource type."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "ProxyResource":{
+      "description":"ARM proxy resource.",
+      "allOf":[
+        {
+          "$ref":"#/definitions/Resource"
+        }
+      ]
+    },
+    "DisasterRecoveryConfigurationProperties": {
+      "properties": {
+        "status": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The status of the disaster recovery configuration."
```

Comment:
Do we have a list known values? so that we can mark it as enum with modelAsString as true.

Reply:
Added the enum. (Also found that the enum for failoverPolicy was incorrect and fixed it.)